lib/utils/report.cjs

```diff
@@ -168,16 +168,16 @@ function isFixApplied({ fix, line, result: { stylelint }, ruleName }) {
 		return false;
 	}
 
-	const hasFix = Boolean(config.fix && !config.rules?.[ruleName][1]?.disableFix);
-	const fixed = hasFix && (config.ignoreDisables || !isDisabled(ruleName, line, disabledRanges));
+	const shouldFix = Boolean(config.fix && !config.rules?.[ruleName][1]?.disableFix);
+	const mayFix = config.ignoreDisables || !isDisabled(ruleName, line, disabledRanges);
```

Comment:
`fixed` is confusing because it implies mutation of the source, which is not what happens on this line.

The conditions only check if fixing is allowed.